engine/class_modules/sc_mage.cpp

```diff
@@ -574,6 +574,7 @@ struct mage_t final : public player_t
     bool heat_shimmer;
     int embedded_splinters;
     int magis_spark_spells;
+    int intuition_BLP_count;
```

Comment:
IIRC BLP is not capitalized in simc source (i.e. it would be `intuition_blp_count`), but that's really just a nitpick.